app/models/form_profiles/va_526ez.rb

```diff
@@ -93,7 +93,10 @@ def initialize_rated_disabilities_information
 
     settings = Settings.lighthouse.veteran_verification.form526
     service = ApiProviderFactory.rated_disabilities_service_provider(
-      user
+      {
+        auth_headers: EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h),
+        icn: user.icn.to_s
```

Comment:
I'm guessing this module/classname reference to EVSS doesn't imply it actually uses EVSS, but it was just built around it.

Since we're moving off of EVSS, that name is no longer helpful or accurate, right? 

Maybe it's beyond scope of this work to update the underlying module and its various references, but otherwise we're contributing to technical debt, no? I wonder if we could at least create a card to properly migrate this module and drop it in the backlog?

Reply:
in this case, it does refer to EVSS itself.
the call to the underlying EVSS service relies on these "`auth_headers`" to be sent, while the other providers do not. 
maybe to be more clear we can call the attribute "`evss_auth_headers"`.

Reply:
We discussed this offline. I now understand that `EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)` is required in the short term because prod is not yet able to identify user identification via `icn` (which Lighthouse is uses). After Lighthouse is moved to prod, this line will be removed. ✅